templates/etc/fastd/fastd.conf.erb

```diff
@@ -12,7 +12,20 @@ hide mac addresses yes;
 include "secret.conf";
 mtu <%= @mesh_mtu %>; # 1492 - IPv{4,6} Header - fastd Header...
 status socket "/var/run/fastd-status.<%= @mesh_code %>.sock";
-include peers from "peers";
+peer group "nodes" {
+  include peers from "peers";
+<% if @peer_limit %>
```

Comment:
Ack, thanks. Fixed in bbff9b7a5e on ffgtso; as this seems not to be wanted anyway, I skip the pull request for now.